packages/dds/tree/.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+    "extends": [
+        "@fluidframework/eslint-config-fluid/eslint7"
+    ],
+    "parserOptions": {
+        "project": ["./tsconfig.json", "./src/test/tsconfig.json"]
+    },
+    "rules": {
+        "@typescript-eslint/no-use-before-define": "off",
+        "@typescript-eslint/strict-boolean-expressions": "off",
+        "keyword-spacing": "off", // Off because it conflicts with typescript-formatter
+        "no-case-declarations": "off",
+        "no-null/no-null": "off",
+        "prefer-arrow/prefer-arrow-functions": "off"
```

Comment:
Can you audit these exceptions to make sure they're truly needed? Once you switch to the standard config (as opposed to the soon to be removed eslint7 one) some of these may be redundant).

Reply:
Thank you for bringing this up. None of those were needed.

Reply:
@tylerbutler - FYI - I had to re-introduce more exceptions. Letting you know in case you want to (re)review them.